### PR TITLE
skip tests Gd perf monitor

### PR DESCRIPTION
**Pull request by gdlex4015**
Added one-off workflow monitoring in Perf test script
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by aweng98 — APPROVED** [review]

File: automation/submission-perf-test.sh [hunk 1]
```diff
@@ -88,7 +89,30 @@ launchSubmission() {
     fi
 }
 
-     # check if user needs a token refresh
+findSubmissionID() {
+    user=$1
+    namespace=$2
+    name=$3
+
+     ACCESS_TOKEN=`docker run --rm -v $WORKING_DIR:/app/populate -w /app/populate broadinstitute/dsp-toolbox python get_bearer_token.py "${user}" "${JSON_CREDS}"`
```
Comment (MatthewBemis) [inline automation/submission-perf-test.sh]: nit: spacing

File: automation/submission-perf-test.sh [hunk 2]
```diff
@@ -88,7 +89,30 @@ launchSubmission() {
...
+monitorSubmission() {
+    user=$1
+    namespace=$2
+    name=$3
+    submissionId=$4
+
+    ACCESS_TOKEN=`docker run --rm -v $WORKING_DIR:/app/populate -w /app/populate broadinstitute/dsp-toolbox python get_bearer_token.py "${user}" "${JSON_CREDS}"`
+
+    submissionStatus=$(curl -X GET --header 'Accept: application/json' --header "Authorization: Bearer $ACCESS_TOKEN" "https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/workspaces/$namespace/$name/submissions/$submissionId" | jq -r '.status')
+    workflowsStatus=$(curl -X GET --header 'Accept: application/json' --header "Authorization: Bearer $ACCESS_TOKEN" "https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/workspaces/$namespace/$name/submissions/$submissionId"  | jq -r '.workflows[] | .status')
```
Comment (MatthewBemis) [inline automation/submission-perf-test.sh:112]: is there only 1 workflow in the submission?

File: automation/submission-perf-test.sh [hunk 3]
```diff
@@ -112,6 +136,28 @@ if [ $ENV = "alpha" ]; then
     launchSubmission hermione.owner@test.firecloud.org aa-test041417 Perf-Test-G-W alex_methods sleep_echo_strings sample_set sample_set6k true "this.samples"
     launchSubmission dumbledore.admin@test.firecloud.org aa-test-042717a test-042717 anuMethods callCacheWDL participant subject_HCC1143 true
 
+    #Monitor the progress of the perf test
+    findSubmissionID dumbledore.admin@test.firecloud.org aa-test-042717a test-042717
+    monitorSubmission dumbledore.admin@test.firecloud.org aa-test-042717a test-042717 $submissionId
```
Comment (MatthewBemis) [inline automation/submission-perf-test.sh:141]: is this stuff only going to be checked for alpha tests?

**Review by MatthewBemis — COMMENTED** [review]
a few minor comments/questions

**Pull request merged**